### PR TITLE
EPRS 6.2.17 and 7.0.1 - New Releases

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

6.2:

- Added 6.2.17 rel notes
- Removed JDK 7 references
- Removed date and added money to the list of restricted types

7.0:

- Removed JDK 7 references
- Added 7.0.1 rel notes

### CODE REVIEW

File: product_docs/docs/eprs/6.2/07_common_operations/11_using_ssl_connections.mdx [hunk 1]
```diff
@@ -261,7 +261,7 @@ The encrypted password must be specified with the `sslTrustStorePassword` config
 Encrypt the password using the xDB Replication Server CLI encrypt command. The following example shows this process encrypting the password contained in file infile.
 
 ```text
-$ export PATH=/usr/lib/jvm/java-1.7.0-openjdk-1.7.0.45.x86_64/jre/bin:$PATH
+$ export PATH=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.45.x86_64/jre/bin:$PATH
```
Comment (mlewandowski-edb) [inline product_docs/docs/eprs/6.2/07_common_operations/11_using_ssl_connections.mdx]: Is "java-1.8.0-openjdk-1.8.0.45.x86_64" correct? Other references (earlier and later) to the Java 8 path say "java-.0-openjdk-1.8.0.322.b06-1.el7_9.x86_64".

File: product_docs/docs/eprs/6.2/eprs_rel_notes/13_eprs_rel_notes_6.2.16.mdx [hunk 2]
```diff
@@ -1,29 +1,11 @@
 ---
-title: "Version 6.2.16"
+title: "Version 6.2.17"
```
Comment (mlewandowski-edb) [inline product_docs/docs/eprs/6.2/eprs_rel_notes/13_eprs_rel_notes_6.2.16.mdx]: The change indicates that this file is related to 6.2.17, but the filename appears to be for 6.2.16 (product_docs/docs/eprs/6.2/eprs_rel_notes/13_eprs_rel_notes_6.2.16.mdx)

File: product_docs/docs/eprs/6.2/eprs_rel_notes/12_eprs_rel_notes_6.2.17.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Version 6.2.16"
```
Comment (mlewandowski-edb) [inline product_docs/docs/eprs/6.2/eprs_rel_notes/12_eprs_rel_notes_6.2.17.mdx]: The filename (product_docs/docs/eprs/6.2/eprs_rel_notes/12_eprs_rel_notes_6.2.17.mdx) indicate that this is meant to be a new document that was meant to be used for release 6.2.17, but this title and remaining content appears to correspond to release 6.2.16.

File: product_docs/docs/eprs/6.2/07_common_operations/11_using_ssl_connections.mdx (hunk 1 above)
Comment (drothery-edb) [reply product_docs/docs/eprs/6.2/07_common_operations/11_using_ssl_connections.mdx]: This is correct and the other instances are wrong. I thought I had caught them all.

File: product_docs/docs/eprs/7.0/eprs_rel_notes/19_eprs_rel_notes_7.0.1.mdx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+title: "Version 7.0.1"
+---
+
+
+New features, enhancements, bug fixes, and other changes in EDB Postgres Replication Server 7.0.1 include the following:
+
+| Type        | Description |
+| ----------- |------------ |
+| Security Fix| Addresses JDBC vulnerability CVE-2022-21724 that has a CVSS score of 8.5, which is a High severity. The PostgreSQL JDBC driver version 42.3.2 that contains the security fix is bundled with Replication Server. If you have a RPM- or Debian-based Replication Server installation, the EDB JDBC driver for Advanced Server, which incorporates the community version 42.3.2, needs to be updated. For example, on RHEL 7 or CentOS 7 distributions, run: `yum update edb-jdbc`. |
+| Bug Fix     | Publication database registration fails for a SSL connection with a non-default SSL client certs/key path. [77621] |
```
Comment (mlewandowski-edb) [inline product_docs/docs/eprs/7.0/eprs_rel_notes/19_eprs_rel_notes_7.0.1.mdx:11]: Is this simply a placeholder for actual bug fix(es) that may have been implemented in 7.0.1.  This fix was made in 6.2.17, but not sure about 7.0.1.  Would need to verify.

**Pull request merged**